README.md

```diff
@@ -127,11 +127,13 @@ If you have geometries in longitude and latitude (EPSG:4326), you can convert to
       (x0, y0, x_max, y_max) = tile_bounds.extent
       x_span = x_max - x0
       y_span = y_max - y0
-      def xy_pairs():
-          for x_merc, y_merc in line:
-              yield (
-                  int((x_merc - x0) * MVT_EXTENT / x_span),
-                  int((y_merc - y0) * MVT_EXTENT / y_span),
+
+      tile_based_coords = []
+      for x_merc, y_merc in line:
```

Comment:
I'm not sure what the `line` object would be at this point, and whether it would support iteration over the points directly, or (as in Shapely) would need `line.coords`.

If we don't know that the code example works, would it be better to remove it?

Reply:
I tested that it executed, so I'm confident that the code itself doesn't have issues with the api like that. But to your point about removing it, we can go that way too.

Reply:
Oh, okay. Cool! If it works, then even better :+1: